src/store/migrations/fix-aes-ctr-counter-issue.js

```diff
@@ -3,7 +3,7 @@ import AES from '../../lib/aes';
 
 export default {
   migrate(state, store) {
-    if (!process.env.IS_MOBILE_DEVICE) return undefined;
+    if (!process.env.IS_MOBILE_DEVICE) return state;
```

Comment:
if the `state` is not a promise, won't this fail again with the same error?

Reply:
This won't fail, because `state` is not `undefined`, so `state.then` is `undefined` but it won't raise an exception.
In the current implementation of migration runner, migration should return a new state or a promise to run a migration. I forgot to return unchanged state in the desktop version of the Base app.

Reply:
Are you making use of the `state` when you return it? Why not make things cleaner and basically make the function return always a promise, instead of creating different `return` values, and avoid this kind of issues.

Reply:
It would be great if you reviewed #205 before merging.
Currently, migrations runner support two types of migrations: that runs in sync and that for some reason should be applied later (like one that fixes AES issue).
If migration returns state instead of a promise, it means that this migration runs in sync and the returned state (modified by the migration) should be passed to further migrations and finally saved to localStorage.
We can make all migrations to return promise, but then we need to define some way to distinguish migrations that run before the start of application and migrations that require some user input.